client/public/index.html

```diff
@@ -3,15 +3,15 @@
   <head>
     <!-- Conditionally include Google Analytics snippet, if targetting production -->
     <% if (process.env.NODE_ENV === 'production') { %>
-    <script async src="https://www.googletagmanager.com/gtag/js?id=G-Y3Y8TPP27F"></script>
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-149759341-1"></script>
```

Comment:
When I open our GA dashboard, I only see `UA-146085874-1` as a tracking ID, not the one here. Am I looking in the wrong place, or not have access to the right GA view?

Reply:
I had copied it from other BC repos since I don't think I have access to that dashboard. But I see that Dunrie also used that ID so I made the switch.